.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,34 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Should we add a path filter here, just like this?

https://github.com/mdn/translated-content/blob/abc2a5d225cf19950e5778479b35a3f4efe3fcc9/.github/workflows/pr-test.yml#L13-L15